### ISSUE TITLE
1.14 Can't start MC from dev env due to crash

### ISSUE THREAD

**Issue by drcrazy** [CottonMC/LibCD#12]
Getting crash while starting MC from dev env with LibCD 1.6.3

---- Minecraft Crash Report ----
// Hey, that tickles! Hehehe!

Time: 11/29/19 12:21 PM
Description: Initializing game

```
java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors!
        at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.logErrors(EntrypointUtils.java:46)
        at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointClient.start(EntrypointClient.java:32)
        at net.minecraft.client.MinecraftClient.init(MinecraftClient.java)
        at net.minecraft.client.MinecraftClient.start(MinecraftClient.java:368)
        at net.minecraft.client.main.Main.main(Main.java:127)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:188)
        at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:131)
        at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:26)
        at net.fabricmc.devlaunchinjector.Main.main(Main.java:84)
        Suppressed: java.lang.NoClassDefFoundError: blue/endless/jankson/api/SyntaxError
                at io.github.cottonmc.libcd.LibCD.onInitialize(LibCD.java:53)
                at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.lambda$logErrors$0(EntrypointUtils.java:39)
                at java.util.ArrayList.forEach(ArrayList.java:1257)
                at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.logErrors(EntrypointUtils.java:37)
                ... 12 more
```

Both in eclipse and via ./gradlew runClient
./gradlew build is fine

Other mods in dev env are here: https://github.com/TechReborn/TechReborn/blob/1.14/build.gradle

Not commited change is update of LibCD version

**Comment by Boundarybreaker** [comment]
Ohh wait, I know what's going on and as usual it's my fault

**Comment by Boundarybreaker** [comment]
tl;dr, LibCD is getting a breaking change in 1.15 but I accidentally updated to that breaking change on the 1.14 branch

**Comment by Boundarybreaker** [comment]
Alright, I'm exporting a new version of 1.6.3 that fixes this! It's being uploaded to CurseForge as we speak.
